nginx/scripts/export_cert_values.sh

```diff
@@ -7,3 +7,4 @@ export NGINX_PUBLIC_CERT="$(cat nginx/certs/server_public.crt)"
 export NGINX_PRIVATE_CERT="$(cat nginx/certs/server_private.key)"
 export NGINX_CLIENT_PUBLIC_CERT="$(cat nginx/certs/client_public.crt)"
```

Comment:
change extensions to .cer as the creation certificate instructions use .cer extension